types/error.example.json

```diff
@@ -1,5 +1,14 @@
 {
   "@type": "Error",
-  "identifier": "1",
-  "message": "Conflicting arrival date"
+  "airline": "Spirit Airline",
+  "errorLevel": "error",
+  "errorType": "RCRenderHandlerError",
+  "errorMessage":"em-cmp-lib-prices__fare-list--mosaic--grid__2.3.29|ghl__to-city__en-us => m.fareRestClient.loadData is not a function",
```

Comment:
Part of the errorMessage and eventCategory could maybe be combined into one field "airModule"